### PR TITLE
add usdefully tips into readme

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
 See [full documentation](docs/README.md)
 
 
+# Usefully tips
```
Comment (jenow) [inline README.md]: It's "Useful"

File: README.md [hunk 2]
```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
...
+
+## Updating kuzzle's containers
+
+When you have already installed an old version of kuzzle, don't forget to update kuzzle's containers with :
```
Comment (jenow) [inline README.md]: "already have"

File: README.md [hunk 3]
```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
...
 
 
+# Useful tips
+
+## Updating kuzzle's containers
+
+When you already have installed an old version of kuzzle, don't forget to update kuzzle's containers with :
+
+```
+    $ docker-compose -f docker-compose/test.yml pull
```
Comment (scottinet) [inline README.md]: I would replace `docker-compose/test.yml` with `<docker-compose-file.yml`

File: README.md [hunk 4]
```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
...
+
+When you already have installed an old version of kuzzle, don't forget to update kuzzle's containers with :
+
+```
+    $ docker-compose -f docker-compose/test.yml pull 
+```
+
+## Updating kuzzle's dependencies 
+
+To ensure if your kuzzle's dependencies are updated, go into kuzzle's containers and run :
```
Comment (scottinet) [inline README.md]: `To ensure that Kuzzle's dependencies are up-to-date, go into a Kuzzle container and run:`

File: README.md [hunk 5]
```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
...
+
+```
+    (host)      $ docker exec -ti dockercompose_kuzzle_1 bash
+    (container) $ npm install
```
Comment (scottinet) [inline README.md]: Maybe we could also add:
`bin/kuzzle.js install`

**Comment by scottinet** [comment]
+1

**Comment by xbill82** [comment]
:+1:

File: README.md [hunk 6]
```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
...
+
+## Updating kuzzle's dependencies 
+
+To ensure that Kuzzle's dependencies are up-to-date, go into a Kuzzle container and run:
+
+```
+    (host)      $ docker exec -ti <docker-compose-file.yml> bash
+    (container) $ npm install
+    (container) $ bin/kuzzle.js install
+```
```
Comment (ballinette) [inline README.md]: better for me: run the command directly without log-in into the container:
$ docker exec -ti <docker-compose-file.yml>  npm install
$ docker exec -ti <docker-compose-file.yml>  bin/kuzzle.js install

**Comment by AnthonySendra** [comment]
In order to keep the main readme very simple, can we move this tips section in installation readme ? And change 
```
Take a look at the installation file for more installation ways. (Manual installation, add fixture, database reset and more)
```
By something like
```
Take a look at the installation file for more installation ways. (Manual installation, update, add fixture, database reset and more)
```

**Comment by AnthonySendra** [comment]
+1

**Comment by jenow** [comment]
+1 merge

**Pull request merged**